Makefile

```diff
@@ -23,4 +24,4 @@ panther_push:
 	docker push $(REGISTRY)/${RL_TEAM_ID}:panther_latest
 
 panther_test:
-	docker run $(REGISTRY)/${RL_TEAM_ID}:panther_latest Combatant/tests/test_panther.sh
+	docker run $(REGISTRY)/${RL_TEAM_ID}:panther_latest Combatant/tests/test_panther.sh "<<PATH TO THE AGENT>>"
```

Comment:
So I guess participants need to edit the Makefile with the path to their agent here?

Would it not be possible just to have `test_pelican.py` and `test_panther.py` import AGENTS_PATH from `combatant.py` (and maybe adds "pelican" or "panther" to the end), so it automatically looks at the same agent that `load_combatant` will load?